src/main/java/core/basesyntax/Shape.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public abstract class Shape {
+    protected Color color;
+    protected double area;
+
+    public abstract double getArea();
+
+    public abstract Color getColor();
+
+    public abstract void draw();
```

Comment:
for common behaviour it's better to use interface (in your case declare methods draw() and getArea() in the interface and then implement it in this class)